docs/sphinx/source/changelog/v2.1.2.rst

```diff
@@ -0,0 +1,15 @@
+**************************
+v2.1.2 (December 20, 2021)
+**************************
+
+Documentation
```

Comment:
nitpick: thinking about the ordering of these sections, I think I'd move "Documentation" down to the bottom and have "bug fixes" at the top